.azure-pipelines/dotnet.yml

```diff
@@ -0,0 +1,78 @@
+# External variables:
```

Comment:
I'm good with this being a separate yml file for now but our engineering team may merge this into the primary yml file at a later point to ensure we can build all the assets in one pipeline so we can publish them together.